### PR TITLE
[FIX] art_craft, **: cleanup and fix res_config_setting

### PR DESCRIPTION
**Pull request by shag-odoo**
**=automobile,certification_organism,software_reseller

This PR fixes the issue of enabling advanced option in pricelist menu in
res_config_setting model and also remove unwanted comment and basic cleanup according
to the standard code structure.

### CODE REVIEW

File: software_reseller/data/ir_ui_menu.xml [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
     </record>
     <record id="licenses_licenese_01632710-57f0-4ec5-9e2e-e975cfbc0406" model="ir.ui.menu">
         <field name="name">Software</field>
         <field name="sequence">1</field>
-        <field name="parent_id" ref="software_reseller.licenses_8482a8f5-a077-46b1-8007-6efe437a9245" />
-        <field name="action" ref="software_reseller.licenese_50da6821-c954-4104-b2bf-1a1a498da4de" />
+        <field name="parent_id" ref="licenses_8482a8f5-a077-46b1-8007-6efe437a9245" />
+        <field name="action" ref="licenese_50da6821-c954-4104-b2bf-1a1a498da4de" />
     </record>
 </odoo>
```
Comment (paso-odoo) [inline software_reseller/data/ir_ui_menu.xml:23]:
```suggestion
</odoo>

```

File: software_reseller/demo/sale_order_line.xml [hunk 2]
```diff
@@ -1,174 +1,174 @@
...
     <record id="sale_order_line_17" model="sale.order.line">
         <field name="name">Business Needs analysis</field>
-        <field name="product_id" ref="software_reseller.product_product_7" />
+        <field name="product_id" ref="product_product_7" />
         <field name="price_unit">6000.0</field>
         <field name="product_uom_qty">5.0</field>
         <field name="product_uom" ref="uom.product_uom_day" />
-        <field name="order_id" ref="software_reseller.sale_order_6" />
+        <field name="order_id" ref="sale_order_6" />
         <!-- <field name="planning_hours_to_plan">40.0</field> -->
```
Comment (paso-odoo) [inline software_reseller/demo/sale_order_line.xml:69]:
```suggestion
```

File: art_craft/data/res_config_settings.xml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <odoo noupdate="1">
-
```
Comment (paso-odoo) [inline art_craft/data/res_config_settings.xml:3]: not a required change

**Review by paso-odoo — CHANGES_REQUESTED** [review]

**Comment by paso-odoo** [comment]
@vava-odoo @pimodoo 

Can you check this cleanup PR?

**Review by paso-odoo — APPROVED** [review]

**Review by vava-odoo — APPROVED** [review]
Thanks @shag-odoo for this. Could you also update the commit title and message?

**Comment by shag-odoo** [comment]
Hello @vava-odoo 
Thanks for the suggestion ,commit message and title has been updated

**Pull request merged**